README.md

```diff
@@ -259,27 +259,25 @@ $ source ~/.venvs/sapp/bin/activate
 (sapp) $ pip3 install -r requirements.txt
 ```
 
-Run the flask server:
+Run the flask server in debug mode:
 ```shell
-(sapp) $ python3 -m sapp.cli server
+(sapp) $ python3 -m sapp.cli server --debug
 ```
 
 Parse static analysis output and save to disk:
 ```shell
 (sapp) $ python3 -m sapp.cli analyze taint-output.json
 ```
 
-If you make any changes to files under `sapp/ui/frontend/*`, you will need to run `npm install` once to install dependencies and `npm run-script build` each time you make changes before running the flask server to see the changes you made reflected:
-
-Installing dependencies:
+Installing dependencies for frontend:
 ```shell
 (sapp) $ cd sapp/ui/frontend && npm install
 ```
 
-Build static files and run the flask server:
+Run the flask server and react app in development mode:
```

Comment:
It seems like we need to run the flask server and `npm run-script start` simultaneously. Could you add a line in the docs to communicate that?

Reply:
Sure

Reply:
@0xedward thanks. How does it look now 🤗?